frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignContent/FieldRow/FieldRowContainer.tsx

```diff
@@ -176,14 +214,29 @@ export const FieldRowContainer = ({
                   colorScheme="secondary"
                   spacing={0}
                 >
+                  <IconButton
+                    aria-label="Close field settings"
+                    icon={<BiCog fontSize="1.25rem" />}
+                    onClick={setToInactive}
+                  />
                   <IconButton
                     aria-label="Duplicate field"
+                    // Fields which are not yet created cannot be duplicated
```

Comment:
Do we want to hide the duplicate button instead if state is creating field?

Reply:
ede3bdd8